src/test/AlienCodex.t.sol

```diff
@@ -2,7 +2,7 @@ pragma solidity ^0.8.10;
 
 import "ds-test/test.sol";
 import "../Ethernaut.sol";
-import "./utils/vm.sol";
+import "forge-std/Vm.sol";
 
 
 contract AlienCodexTest is DSTest {
```

Comment:
since vm is a standard library of `forge-std/Test.sol` so basically it can be shorter like this:
```suggestion
import "forge-std/Test.sol";
import "../Ethernaut.sol";

contract AlienCodexTest is Test {
```